src/Authoring/AuthoringConsumptionTest/AuthoringConsumptionTest.vcxproj

```diff
@@ -23,7 +23,6 @@
     <ProjectGuid>{0212a7c5-8d3f-443c-9ebc-1f28091fdf88}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>AuthoringConsumptionTest</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.19592.0</WindowsTargetPlatformVersion>
```

Comment:
Is this part of removing the workaround? Or just something not needed anymore?

Reply:
removing duplication of that property - vcxproj's should pick it up from the central definition in src\directory.build.props